addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -555,7 +555,7 @@ export class Link extends Component {
         ];
         const keptClasses = this.state.iniClassName.split(' ').filter(className => classesToKeep.includes(className));
         const allBtnColorPrefixes = /(^|\s+)(bg|text|border)(-[a-z0-9_-]*)?/gi;
-        const allBtnClassSuffixes = /(^|\s+)btn(-[a-z0-9_-]*)?/gi;
+        const allBtnClassSuffixes = /(^|\s+)btn(-[a-z0-9_-]*)?\b/gi;
```

Comment:
That change is because the old selector removed the "btn" in the "btn_cta" class, which it allegedly should not.